admin/class-metabox.php

```diff
@@ -33,19 +30,18 @@ public function __construct() {
 	 * the main meta box definition array in the class WPSEO_Meta() as well!!!!
 	 */
 	public static function translate_meta_boxes() {
-		self::$meta_fields['general']['snippetpreview']['title'] = __( 'Snippet Preview', 'wordpress-seo' );
+		self::$meta_fields['general']['snippetpreview']['title'] = __( 'Snippet', 'wordpress-seo' );
```

Comment:
Yes, that is correct. This was done because the snippet preview has to be called snippet from now on. So it is the way it should be.